src/lib.rs

```diff
@@ -153,10 +157,20 @@ pub enum Brightness {
     Decrease
 }
 
+#[derive(Clone, ValueEnum, Debug)]
+#[repr(u8)]
+pub enum Module {
+    Modpad,
+    Down,
+    Left,
+    Right
+}
```

Comment:
As underlying code is depending on values assigned to individual *enum* kinds and also to improve code readability. I would suggest to explicitly assign value to each kind. Suggestion below should be correct, but it is not tested.

```suggestion
#[derive(Clone, ValueEnum, Debug)]
#[repr(u8)]
pub enum Module {
    Modpad = 0x00,
    Down = 0x01,
    Left = 0x02,
    Right = 0x03,
}
```